src/index.js

```diff
@@ -1,14 +1,14 @@
 require('setimmediate')
 
-var createHashHistory = require('history/lib/createHashHistory')
+// var createHashHistory = require('history/lib/createHashHistory')
 var React = require('react')
 var {render} = require('react-dom')
 var Router = require('react-router/lib/Router')
-var useRouterHistory = require('react-router/lib/useRouterHistory')
-var withScroll = require('scroll-behavior/lib/withStandardScroll')
+// var useRouterHistory = require('react-router/lib/useRouterHistory')
+// var withScroll = require('scroll-behavior/lib/withStandardScroll')
 
 var routes = require('./routes')
 
-var history = withScroll(useRouterHistory(createHashHistory)())
+// var history = withScroll(useRouterHistory(createHashHistory)())
```

Comment:
When I was working on these updates, a number of the pieces here (including scroll-behavior) changed their APIs and it became super tricky to get them working again. Or at least I thought. I've pushed some commits restoring the functionality that was there before using the latest react-router scroll middleware.